buildenv/jenkins/openjdk_tests

```diff
@@ -151,7 +151,15 @@ if (JDK_VERSIONS.size() > 1 || JDK_IMPLS.size() > 1 || PLATFORMS.size() >1 || PL
 } else {
     if (PLATFORM_MAP.containsKey(params.PLATFORM)) {
         SPEC = PLATFORM_MAP[params.PLATFORM]["SPEC"]
-        LABEL = params.LABEL ? params.LABEL : PLATFORM_MAP[params.PLATFORM]["LABEL"]
+        if (params.LABEL) {
+                  LABEL = params.LABEL
+                } else {
+                      LABEL = PLATFORM_MAP[params.PLATFORM]["LABEL"]
+                        def perfLabel = LABEL.minus("ci.role.test&&").concat("&&ci.role.perf")
```

Comment:
would prefer to move this `def perfLabel ` inside the if condition.

```
LABEL = PLATFORM_MAP[params.PLATFORM]["LABEL"]
if (params.BUILD_LIST.contains("perf") && areNodesWithLabelOnline(perfLabel)) { 
   def perfLabel = LABEL.minus("ci.role.test&&").concat("&&ci.role.perf")
   LABEL = perfLabel 
}
```

Reply:
@sophia-guo the above will not work. `areNodesWithLabelOnline() ` uses `perfLabel `

Reply:
yes, didn't notice it. Probably the following code? The main point is perfLabel is only related with perf so moving under 'perf' condition may be better. 

```
LABEL = PLATFORM_MAP[params.PLATFORM]["LABEL"] 
if (params.BUILD_LIST.contains("perf") {
    def perfLabel = LABEL.minus("ci.role.test&&").concat("&&ci.role.perf")
    if (areNodesWithLabelOnline(perfLabel))
         {  LABEL = perfLabel }
}
```

Reply:
@sophia-guo changes made ...grinder link https://ci.adoptopenjdk.net/job/Grinder/2815/